runner/src/internal.rs

```diff
@@ -47,6 +47,10 @@ pub enum Command {
     RunTests,
     RunTestsTsan,
     RunCi,
+    #[structopt(
+        about = "generate bash completion with: alias runner=./scripts/runner && source <(runner completion)"
```

Comment:
nit: the help message seems to mix up describing things that are separate from / independent of this, maybe just "generate bash completion script, suitable for feeding into `source`" or somesuch?

Reply:
Yeah I was trying to have the binary self-describe how to generate the completions in practice, instead of adding it to some other external piece of documentation, but perhaps that was not a great idea. I have now added an explicit paragraph in the development documentation instead, at least we can point people there to start with.